### PR TITLE
cargo-add, cargo-bench, cargo-build, cargo-init: add French translation

### PR DESCRIPTION
**Pull request by trouyer**
see #13570 

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages.fr/common/cargo-add.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Ajoute une version spécifique d'une dépendance :
+
+`cargo add {{dépendance}}@{{version}}`
+
+- Ajoute une dépendance et active une ou plusieures fonctionnalités :
+
+`cargo add {{dépendance}} --features {{fonctionnalité_1}},{{fonctionnalité_2}}`
+
+- Ajoute une dépendance optionnelle, qui sera exposée comme une fonctionnalité du crate:
```
Comment (noraj) [inline pages.fr/common/cargo-add.md:18]:
```suggestion
- Ajoute une dépendance optionnelle, qui sera exposée comme une fonctionnalité du crate :
```

File: pages.fr/common/cargo-add.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Ajoute un crate local en tant que dépendance :
+
+`cargo add --path {{chemin/vers/crate}}`
+
+- Ajoute une dépendance de développement ou de compilation :
+
+`cargo add {{dépendance}} --{{dev|build}}`
+
+- Ajoute une dépendance avec toutes les fonctionnalités par défaut désactivées:
```
Comment (noraj) [inline pages.fr/common/cargo-add.md:30]:
```suggestion
- Ajoute une dépendance avec toutes les fonctionnalités par défaut désactivées :
```

File: pages.fr/common/cargo-bench.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# cargo bench
+
+> Compile et exécute des benchmarks.
+> Plus d'informations : <https://doc.rust-lang.org/cargo/commands/cargo-bench.html>.
+
+- Execute tous les benchmarks d'un paquet:
```
Comment (noraj) [inline pages.fr/common/cargo-bench.md:6]:
```suggestion
- Execute tous les benchmarks d'un paquet :
```

File: pages.fr/common/cargo-bench.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`cargo bench`
+
+- Ne pas arréter quand un benchmark échoue:
```
Comment (noraj) [inline pages.fr/common/cargo-bench.md:10]:
```suggestion
- Ne pas arréter quand un benchmark échoue :
```

File: pages.fr/common/cargo-bench.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`cargo bench --no-fail-fast`
+
+- Compile, mais n'exécute pas les benchmarks:
```
Comment (noraj) [inline pages.fr/common/cargo-bench.md:14]:
```suggestion
- Compile, mais n'exécute pas les benchmarks :
```

File: pages.fr/common/cargo-bench.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`cargo bench --no-run`
+
+- Exécute le benchmark spécifié:
```
Comment (noraj) [inline pages.fr/common/cargo-bench.md:18]:
```suggestion
- Exécute le benchmark spécifié :
```

File: pages.fr/common/cargo-bench.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`cargo bench --bench {{benchmark}}`
+
+- Exécute les benchmarks avec un profile spécifique (défaut: `bench`):
```
Comment (noraj) [inline pages.fr/common/cargo-bench.md:22]:
```suggestion
- Exécute les benchmarks avec un profile spécifique (défaut : `bench`) :
```

File: pages.fr/common/cargo-bench.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+`cargo bench --profile {{profile}}`
+
+- Exécute les benchmarks sur tous les examples cibles :
```
Comment (noraj) [inline pages.fr/common/cargo-bench.md:26]:
```suggestion
- Exécute les benchmarks sur tous les exemples cibles :
```

File: pages.fr/common/cargo-build.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+# cargo build
+
+> Compile un paquet local et toutes ses dépendances.
+> Plus d'informations : <https://doc.rust-lang.org/cargo/commands/cargo-build.html>.
+
+- Compile un ou plusieurs paquets définis dans le manifeste `Cargo.toml` dans le dossier local :
+
+`cargo build`
+
+- Compile les artefacts avec le mode release, avec des optimizations :
```
Comment (noraj) [inline pages.fr/common/cargo-build.md:10]:
```suggestion
- Compile les artefacts avec le mode publication, avec des optimisations :
```

File: pages.fr/common/cargo-init.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
+# cargo init
+
+> Crée un nouveau paquet Cargo.
+> Equivalent de `cargo new`, mais spécifier un dossier est optionnel.
```
Comment (noraj) [inline pages.fr/common/cargo-init.md:4]:
```suggestion
> Équivalent de `cargo new`, mais où spécifier un dossier est optionnel.
```

File: pages.fr/common/cargo-init.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+> Plus d'informations : <https://doc.rust-lang.org/cargo/commands/cargo-init.html>.
+
+- Initilise un projet Rust binaire dans le dossier current :
```
Comment (noraj) [inline pages.fr/common/cargo-init.md:7]:
```suggestion
- Initialise un projet Rust binaire dans le dossier courant :
```

File: pages.fr/common/cargo-init.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+
+`cargo init`
+
+- Initilise un projet de binaire Rust dans le dossier spécifié :
+
+`cargo init {{chemin/vers/dossier}}`
+
+- Initialise un projet de bibliothèque Rust  dans le dossier spécifié :
```
Comment (noraj) [inline pages.fr/common/cargo-init.md:15]:
```suggestion
- Initialise un projet de bibliothèque Rust dans le dossier spécifié :
```

File: pages.fr/common/cargo-init.md [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+
+`cargo init --lib`
+
+- Initialise un dépot de système de gestion de version dans le dossier du projet (defaut : `git`) :
```
Comment (noraj) [inline pages.fr/common/cargo-init.md:19]:
```suggestion
- Initialise un dépôt de système de gestion de version dans le dossier du projet (défaut : `git`) :
```

File: pages.fr/common/cargo-init.md [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+
+`cargo init --vcs {{git|hg|pijul|fossil|none}}`
+
+- Défini le nom du paquet (defaut: nom du dossier):
```
Comment (noraj) [inline pages.fr/common/cargo-init.md:23]:
```suggestion
- Défini le nom du paquet (défaut : nom du dossier):
```

**Review by noraj — CHANGES_REQUESTED** [review]
Thanks for your contribution, please fix the following issues.

**Comment by trouyer** [comment]
> Thanks for your contribution, please fix the following issues.

Thanks Noraj, it's fixed

**Review by noraj — APPROVED** [review]

**Pull request merged**